cms/admin/permissionadmin.py

```diff
@@ -51,8 +51,9 @@ def get_queryset(self, request):
         # can see only permissions for users which are under him in tree
 
         # here an exception can be thrown
+        model = self.model
```

Comment:
In fact I initially started with the qs = self.model…. just to see that another portion of code had model = self.model.
Finally I did not change it all as I thought it did not make real sense :)

> Le 26 juil. 2015 à 22:14, Iacopo Spalletti <notifications@github.com> a écrit :
> 
> In cms/admin/permissionadmin.py <https://github.com/divio/django-cms/pull/4316#discussion_r35496896>:
> 
> > @@ -51,8 +51,9 @@ def get_queryset(self, request):
> >          # can see only permissions for users which are under him in tree
> >  
> >          # here an exception can be thrown
> > +        model = self.model
> Why using a different code style instead of doin qs = self.model... like for PageAdmin?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/divio/django-cms/pull/4316/files#r35496896>.
>